subsys/net/lib/dns/dispatcher.c

```diff
@@ -260,6 +260,10 @@ int dns_dispatcher_register(struct dns_socket_dispatcher *ctx)
 				goto out;
```

Comment:
In the dns dispatcher commit:
* typo in commit subject s/entires/entries/
* wasn't it that the `CONFIG_DNS_RESOLVER_MAX_SERVERS` was smaller than the servers in dhcp msg that triggered the error

Reply:
Fixed the typo. 

> wasn't it that the CONFIG_DNS_RESOLVER_MAX_SERVERS was smaller than the servers in dhcp msg that triggered the error
No, if CONFIG_DNS_RESOLVER_MAX_SERVERS was smaller, DNS code would simply ignore redundant servers. The case was when CONFIG_DNS_RESOLVER_MAX_SERVERS was larger than the number of servers, where some of the server entries were left unset (I. e. socket FD was left -1)